src/components/ErrorBoundary/BaseErrorBoundary.js

```diff
@@ -40,10 +42,13 @@ class BaseErrorBoundary extends React.Component {
         BootSplash.hide({fade: true});
     }
 
+    onRefresh() {
```

Comment:
Functions should be named for what they do, not the event they handle:

```suggestion
    clearError() {
```